src/components/table/DatasetTable.jsx

```diff
@@ -71,6 +71,11 @@ class DatasetTable extends React.PureComponent {
         property: 'storage.array',
         render: (row) => Array.from(new Set(row.storage.map((s) => s.region))).join(', '),
       },
+      {
+        label: 'Cloud Platform',
+        property: 'storage.array',
+        render: (row) => Array.from(new Set(row.storage.map((s) => s.cloudPlatform))).join(', '),
```

Comment:
Do we support datasets or snapshots with more than one cloud platform?

Reply:
Currently, yes. Since we've been slowly porting functionality from GCP to Azure, we could have datasets with both cloud providers. Once we finish up the azure work, there won't be any combined cloud platform datasets.